java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -2315,14 +2322,14 @@ public int add(AggregationOverWindow op, int index) {
   }
 
   /**
-   * Class representing aggregate operations
+   * Class representing groupby operations
    */
-  public static final class AggregateOperation {
+  public static final class GroupByOperation {
```

Comment:
I am a little concerned that this is a breaking change.  It is not the end of the world and I do think it is a good change. I just want to be sure that if a patch is needed to make the Spark plugin work that it too is ready to go. I don't think it is needed because it looks like Scala will auto detect the type when compiling, but I want to check.

Reply:
I have already verified it locally, no patch is needed for the Spark plugin to work. Because no explicit class reference is used in plugin, i think.